site/content/en/references/kustomize/cmd/build/_index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "kustomize build"
+linkTitle: "build"
+type: docs
+weight: 2
+description: >
+    Print configuration per contents of kustomization.yaml
+---
```

Comment:
We don't need to turn this PR into a full solution to https://github.com/kubernetes-sigs/kustomize/issues/3464, but since this is the main command let's add a quick sentence to this one. How about:

> `kustomize build` is Kustomize's primary command. It recursively builds (aka _hydrates_) the kustomization.yaml you point it to, resulting in a set of Kubernetes resources ready to be deployed. See the [Kustomization File reference](https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/) for details on how this works and what a kustomization.yaml can include.